composer.json

```diff
@@ -5,6 +5,7 @@
   "require": {
     "php": ">=5.4",
     "symfony/symfony": "~2.8.6",
+    "defuse/php-encryption": "~2.0.1",
```

Comment:
Since we already require `ircmaxell/random-lib` (see [here](https://github.com/julienbourdeau/PrestaShop/blob/e883fa9cd399807bf2638e853906a0994a8abb67/composer.json#L30-L30)) What about using it? From my understanding, it does pretty much the same.

This would avoid adding another lib.